kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -119,44 +117,61 @@
       LEARNER: () => UserKinds.LEARNER,
       COACH: () => UserKinds.COACH,
       ADMIN: () => UserKinds.ADMIN,
-      statusMessage() {
-        if (this.errorMessage) {
-          return this.errorMessage;
-        } else if (this.confirmationMessage) {
-          return this.confirmationMessage;
+      usernameAlreadyExists() {
+        const index = this.users.findIndex(user => user.username === this.username);
+        if (index === -1) {
+          return false;
         }
-        return false;
+        return true;
+      },
+      usernameNotAlphaNum() {
+        if (this.username === '') {
+          return false;
+        }
+        const re = /^\w+$/g;
+        return !re.test(this.username);
```

Comment:
Gotcha. I scanned it wrong. It might be more readable if you combined the two conditions:

```
return (this.username !== '') && !re.test(this.username)
```

To me, it reads as "user name is not blank and it is not alphanumeric".

Also, in general, it's sometimes better to defined predicates in terms of the positive case (e.g. `usernameIsAlphaNum`) and then you would just negate it inside `usernameInvalid`